payment_paypal/indico_payment_paypal/controllers.py

```diff
@@ -84,14 +88,22 @@ def _verify_business(self):
         return False
 
     def _verify_amount(self):
-        expected_amount = self.registration.price
+        paypal_fixed_fee = float(current_plugin.event_settings.get(self.registration.registration_form.event, 'paypal_fixed_fee'))
+        paypal_percent_fee = float(re.findall(r'^([0-9]+(\.[0-9]+)?)(%)?', current_plugin.event_settings.get(self.registration.registration_form.event, 'paypal_percent_fee'))[0][0])
+
+        expected_amount = float(self.registration.price)
+
+        current_plugin.logger.info("Checking PayPal payment applying fees to expected amount: %s fixed: %s percent: %s", expected_amount, paypal_fixed_fee, paypal_percent_fee)
+
+        expected_amount_with_paypal_fees = round (( expected_amount + paypal_fixed_fee ) / ( 1 - ( paypal_percent_fee / 100 )), 2)
```

Comment:
please keep lines of code up to 121 characters maximum for readability, and remove unnecessary blank lines

Reply:
should be done